.azure/pipelines/examples-test.yml

```diff
@@ -3,6 +3,40 @@ pr:
 variables:
   API_KEY: $(apiKey)
 jobs:
+  - job: conftest
+    displayName: Conftest
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - bash: >
+          INSTALL_LOCATION="$(Build.SourcesDirectory)/conftest"
+
+          mkdir -p ${INSTALL_LOCATION}
+
+          curl -sL -o /tmp/conftest.tar.gz
+          "https://github.com/open-policy-agent/conftest/releases/download/v0.30.0/conftest_0.30.0_Linux_x86_64.tar.gz"
+
+          tar -xzf /tmp/conftest.tar.gz -C ${INSTALL_LOCATION}
+
+          chmod +x ${INSTALL_LOCATION}/conftest
+
+          echo "##vso[task.setvariable variable=PATH;]$(PATH):${INSTALL_LOCATION}"
+        displayName: Setup Conftest
+      - task: InfracostSetup@0
+        displayName: Setup Infracost
+        inputs:
+          apiKey: $(apiKey)
+      - bash: >-
+          infracost breakdown --path=examples/conftest/code/plan.json
+          --format=json --out-file=/tmp/infracost.json
+        displayName: Run Infracost
+      - bash: >-
+          infracost output --path=/tmp/infracost.json
+          --format=azure-repos-comment --show-skipped
+          --out-file=/tmp/infracost_comment.md
+        displayName: Generate Infracost comment
+      - bash: conftest test --policy examples/conftest/policy /tmp/infracost.json
```

Comment:
@hugorut Do you think we should keep the same approach by using `##vso[task.logissue type=error` output here?

Reply:
hmm no - I prefer this approach. The `##vso[task.logissue type=error]` is a workaround as we need to fail the task output in a boolean way. I like that `conftest test` has this already built in.